discord/guild.py

```diff
@@ -1169,6 +1170,51 @@ def _create_channel(self, name, overwrites, channel_type, category=None, **optio
 
         return result
 
+    async def fetch_all_custom_emojis(self):
+        """|coro|
+
+        Retrieves all custom :class:`Emoji`s from the guild.
+
+        Raises
+        ---------
+        HTTPException
+            An error occurred fetching the emojis.
+
+        Returns
+        --------
+        List[:class:`Emoji`]
+            The retrieved emojis.
+        """
+        data = await self._state.http.get_all_custom_emojis(self.id)
+
+        emojis = [Emoji(guild=self, state=self._state, data=d) for d in data]
+
+        return emojis
+
+    async def fetch_custom_emoji(self, emoji_id):
+        """|coro|
+
+        Retrieves a custom :class:`Emoji` from the guild.
+
+        Parameters
+        -------------
+        emoji_id: :class:`int`
+            The emoji's ID.
+
+        Raises
+        ---------
+        HTTPException
+            An error occurred fetching the emoji.
```

Comment:
Surely there are more exceptions that are raised than just the catch-all `HTTPException`.